### PR TITLE
FIX: Left nav link for GitHub repos was broken, fixed

### PR DESCRIPTION
**Pull request by connor-leech**
#### Changelog

**Changed**

Left nav link for GitHub repos was broken. I fixed it.

### CODE REVIEW

File: packages/addons-website/src/components/WebsiteSideNav/WebsiteSideNav-story.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ storiesOf('Website side nav', module)
             <SideNavLink
               icon={<Launch16 />}
               href="javascript:void(0)"
-              to="/resources#github"
+              to="/resources/#github-repos"
```
Comment (aledavila) [inline packages/addons-website/src/components/WebsiteSideNav/WebsiteSideNav-story.js]: Was only the light theme supposed to change ? I noticed the dark variation still says /#github

Comment (alisonjoseph) [reply packages/addons-website/src/components/WebsiteSideNav/WebsiteSideNav-story.js]: ohh thats why the link was wrong on the homepage. yea we need to update both

Comment (connor-leech) [reply packages/addons-website/src/components/WebsiteSideNav/WebsiteSideNav-story.js]: Ahhhh, thank you for catching that.

**Comment by alisonjoseph** [comment]
you update the demo links in the addons storybook, not the actual left nav. 
https://github.com/carbon-design-system/carbon-website/blob/master/src/components/LeftNav/LeftNav.js#L85 is what you need to update for the site, sorry I just noticed.

**Comment by alisonjoseph** [comment]
closing in favor of https://github.com/carbon-design-system/carbon-website/pull/1409

**Pull request closed without merging**